### PR TITLE
add more context on loading & writing logs

### PR DESCRIPTION
**Pull request by spoonincode**

### CODE REVIEW

**Review by heifner — APPROVED** [review]
cerr yuck, but I guess it can't be helped.

File: src/pinnable_mapped_file.cpp [hunk 1]
```diff
@@ -272,13 +272,13 @@ void pinnable_mapped_file::save_database_file() {
...
       if(time(nullptr) != t) {
          t = time(nullptr);
-         std::cerr << "              " << offset/(_file_mapped_region.get_size()/100) << "% complete..." << std::endl;
+         std::cerr << "CHAINBASE: Writing \"" << _database_name << "\" database file, " << offset/(_file_mapped_region.get_size()/100) << "% complete..." << std::endl;
       }
    }
-   std::cerr << "           Syncing buffers..." << std::endl;
+   std::cerr << "CHAINBASE: Writing \"" << _database_name << "\" database file, flushing buffers..." << std::endl;
    if(_file_mapped_region.flush(0, 0, false) == false)
       std::cerr << "CHAINBASE: ERROR: syncing buffers failed" << std::endl;
```
Comment (linh2931) [inline src/pinnable_mapped_file.cpp:280]: Do you want to change `syncing buffers` to `Writing`?

**Comment by spoonincode** [comment]
yeah out of time to something better before 3.2, but stderr is the default for nodeos so it's kinda consistent in that regard

File: src/pinnable_mapped_file.cpp (hunk 1 above)
Comment (spoonincode) [reply src/pinnable_mapped_file.cpp:280]: yes true probably should be changed to "flushing" to match the change in wording above it

**Review by linh2931 — APPROVED** [review]

**Pull request merged**